modules/authentication/proc-enabling-authentication-with-microsoft-azure.adoc

```diff
@@ -70,7 +70,37 @@ signInPage: microsoft # <3>
 <2> Apply the Microsoft Azure credentials configured in your {product-short} secrets.
 <3> Set the Microsoft Azure provider as your {product-short} sign-in provider.
 
-.. Optional: Consider adding following optional fields:
+`environment: production`::
+Mark the environment as production to hide the **Guest** login in the {product-short} home page.
+
+`clientId`, `clientSecret` and `tenantId`::
+Use the {product-short} application information that you have created in Microsoft Azure and configured in OpenShift as secrets.
+
+`signInPage: microsoft`::
+Enable the Microsoft Azure provider as default sign-in provider.
+
+Optional: Consider adding following optional fields:
+
+`dangerouslyAllowSignInWithoutUserInCatalog: true`::
++
+To enable authentication without requiring to provision users in the {product-short} software catalog.
++
+WARNING: Use `dangerouslyAllowSignInWithoutUserInCatalog` to explore {product-short} features, but do not use it in production.
++
+.`{my-app-config-file}` fragment with optional field to allow authenticating users absent from the software catalog
+[source,yaml]
+----
+auth:
+  environment: production
+  providers:
+    microsoft:
+      production:
+        clientId: ${AUTH_AZURE_CLIENT_ID}
+        clientSecret: ${AUTH_AZURE_CLIENT_SECRET}
+        tenantId: ${AUTH_AZURE_TENANT_ID}
+signInPage: microsoft
+dangerouslyAllowSignInWithoutUserInCatalog: true
+----
```

Comment:
Do not revert changes made in https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/982/files

```suggestion
```